install_config/upgrading/manual_upgrades.adoc

```diff
@@ -1156,133 +1155,44 @@ deployed. For example, if the project is named *logging*:
 $ oc project logging
 ----
 
-. Scale down your Fluentd instances to 0:
+. Recreate the deployer templates for service accounts and running the deployer
 +
 ----
-$ oc scale dc/logging-fluentd --replicas=0
+$ oc apply -n openshift -f \
+    https://raw.githubusercontent.com/openshift/origin-aggregated-logging/master/deployer/deployer.yaml
```

Comment:
Is there an on-disk version of this file we can reference here for OCP? `/usr/share/openshift/examples/infrastructure-templates/enterprise/logging-deployer.yaml`?